aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,171 @@
-﻿.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
 
-.. _bundling-minification-index:
+By `Rick Anderson`_, `Erik Reitan`_, `Daniel Roth`_
 
-|stub-icon| Bundling and Minification
-=====================================
+Bundling and minification are two techniques you can use in ASP.NET to improve page load performance for your web application. Bundling combines multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller payloads. Used together, bundling and minification improves load time performance by reducing the number of requests to the server and reducing the size of the requested assets (such as CSS and JavaScript files).
 
-.. include:: /../common/stub-notice.txt
+This article explains the benefits of using bundling and minification, including how these features can be used with ASP.NET 5 applications. 
 
-.. _issue: https://github.com/aspnet/Docs/issues/101
\ No newline at end of file
+.. contents:: In this article:
+  :local:
+  :depth: 1
+
+Overview
+--------
+
+In ASP.NET Core apps, you bundle and minify the client-side resources during design-time using third party tools, such as :doc:`Gulp <using-gulp>` and :doc:`Grunt <using-grunt>`. By using design-time bundling and minification, the minified files are created prior to the application’s deployment. Bundling and minifying ahead of time provides the advantage of fewer moving parts and reduced server load. However, it’s important to recognize that design-time bundling and minification increases build complexity and only works with static files.
```

Comment:
>>Bundling and minifying ahead of time provides the advantage of fewer moving parts and reduced server load.
What does "ahead of time" and "fewer moving parts" mean - I don't think these cliches translate.
Consider
Bundling and minifying before deployment provides the advantage reduced server load {maybe something about less chance of a problem doing B&M on the server}